perfkitbenchmarker/disk.py

```diff
@@ -18,14 +18,62 @@
 """
 
 import abc
+import logging
 
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import resource
 
+FLAGS = flags.FLAGS
+
+
+# These are the (deprecated) old disk type names
 STANDARD = 'standard'
 REMOTE_SSD = 'remote_ssd'
 PIOPS = 'piops'  # Provisioned IOPS (SSD) in AWS
```

Comment:
It looks like Alicloud uses it too, so it's not just AWS any more. I was not expecting this.